op_test.sh

```diff
@@ -4,11 +4,21 @@
 # and executes that - allows for disconnected testing from the easy-rsa
 # repo with TravisCI.
 
-curl -O 'https://raw.githubusercontent.com/OpenVPN/easyrsa-unit-tests/master/easyrsa-unit-tests.sh'
+# EG: Set in Travis: CUSTOM CONFIG
+# env: ERSA_UTEST_CURL_TARGET="tincantech/easyrsa-unit-tests"
+# env: ERSA_UTEST_VERB_LEVEL="-v" (or "-vv" -t/-tv no delay)
+
+export ERSA_UTEST_CURL_TARGET="${ERSA_UTEST_CURL_TARGET:-OpenVPN/easyrsa-unit-tests}"
+
+curl -O "https://raw.githubusercontent.com/$ERSA_UTEST_CURL_TARGET/master/easyrsa-unit-tests.sh"
+EXIT_CODE=1
 
 if [ -e "easyrsa-unit-tests.sh" ];
 then
-	sh easyrsa-unit-tests.sh -v
+	EXIT_CODE=0
+	ERSA_UTEST_VERB_LEVEL="${ERSA_UTEST_VERB_LEVEL:-"-t"}"
+	sh easyrsa-unit-tests.sh "$ERSA_UTEST_VERB_LEVEL" || EXIT_CODE=1
+	rm easyrsa-unit-tests.sh
 fi
 
-rm easyrsa-unit-tests.sh
```

Comment:
the unit tests should be cleaned up, not sure why you're preventing that here.

Reply:
I'm not preventing anything, I am fixing your broken `op_test.sh`